### PR TITLE
fix: emotes ownership

### PR DESCRIPTION
**Pull request by pedrotambo**

### CODE REVIEW

File: lambdas/src/utils/SmartContentClient.ts [hunk 1]
```diff
@@ -112,8 +112,8 @@ export class SmartContentClient implements ContentAPI {
       try {
         const fetcher = new Fetcher()
         await fetcher.fetchJson(`${SmartContentClient.INTERNAL_CONTENT_SERVER_URL}/status`, {
-          attempts: 6,
-          waitTime: '10s'
+          attempts: 1,
```
Comment (agusaldasoro) [inline lambdas/src/utils/SmartContentClient.ts:115]: should we undo this change?

Comment (pedrotambo) [reply lambdas/src/utils/SmartContentClient.ts]: done

**Review by agusaldasoro — APPROVED** [review]
